charts/flyte-core/values.yaml

```diff
@@ -171,7 +171,7 @@ flytescheduler:
   # -- Annotations for Flytescheduler pods
   podAnnotations: {}
   # -- Additional Flytescheduler container environment variables
-  podEnv: {}
+  podEnv: []
```

Comment:
These are mapped to arrays, so it gives a warning when you set them in the values.yaml.